src/graphics/render_window.rs

```diff
@@ -190,6 +324,21 @@ impl RenderWindow {
     /// has been done for the current frame, in order to show
     /// it on screen.
     ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # // Create a new window
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// while window.is_open() {
+    ///     // Draw something
+    ///     window.display();
```

Comment:
You should at least clear with a color here, or else it will just display garbage.

Reply:
Done